docs/guide/events-guide.md

```diff
@@ -1,40 +1,48 @@
+You can create Events and attach them to any particular Monitoring, Logs, or Experience App, or dashboard. It's great for leaving notes alongside auto-generated data. It supports markdown format and provides a perfect way to provide human input which can later be used for presentations and reminders. 
 
-#### Events Histogram
+### Events Histogram
 
-Events Histogram provides visual insight into events timeseries and can be easily filtered using time picker component shortcuts as well as zoom and shift chart tools. 
+The **Events Histogram** is a timeline of Events time-series data. It can easily be **filtered using the time-picker** component shortcuts, as well as zoom and shift chart tools.
 
 ![Sematext Cloud Events Histogram Component](https://sematext.com/docs/images/guide/alerts-and-events/add-events-component-time-series.png "Sematext Cloud Events Histogram Component")
 
-#### Events Stream
+### Events Stream
 
-On the other hand, Event Stream table component provides a list view of all events that happened across your IT infrastructure. Whether monitoring or logging alert in Apache or Cassandra app was triggered, event stream view centralizes all of them in a single table. It is unified way to filter and pinpoint issues across multiple IT system administration dimensions. 
+The **Event Stream component** is a **list view of all Events** that occurred across your infrastructure. It centralizes all of them in a single table, and is a **unified way to filter and pinpoint issues** across different parts of your infrastructure. 
 
 ![Sematext Cloud Events Stream Component](https://sematext.com/docs/images/guide/alerts-and-events/event-stream-component.png "Sematext Cloud Events Stream Component")
 
-Beyond events that you want to see as part of your operations intelligence think about events that matter to your team or your organization in general. Such "business events" can be shipped to Sematext, too.
-
-Besides being shown in the UI events are also exposed via a REST API that lets you post, retrieve, and search your events. This REST API matches the Elasticsearch API, so you can use any Elasticsearch tool or client to post, get, and search events.
-
-[Register](https://apps.sematext.com/ui/registration) for free or [Login](https://apps.sematext.com/ui/login/) into Sematext IT systems monitoring platform to get started and create your logs app. Upload your logs from all your servers to our centralized log management solution with Elasticsearch API and integrated Kibana, and experience the first true Hosted ELK Stack. 
+Apart from what you see here, you can **send** your own **custom events** to the Sematext Events Stream. Our Events feature is **exposed via a REST API** that lets you post, retrieve, and search your events. This REST API matches the Elasticsearch API, so you can use any Elasticsearch tool or client to post, get, and search for events.
```

Comment:
And can we link the reference to the API to some place where we have an example of a call to this API?

Reply:
added link to https://sematext.com/docs/events/#adding-events